### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by alirizagoksu** [d-max/spots-dialog#24]
at dmax.dialog.SpotsDialog.setMessage(SpotsDialog.java:74) 
```
@Override
public void setMessage(CharSequence message) {
        ((TextView) findViewById(R.id.dmax_spots_title)).setText(message);
}
```
if you set custom message after create dialog but not show dialog yet, it throws NullPointerException. Because it isn't inflated yet. This method should be like that

```
@Override
public void setMessage(CharSequence message) {
       this.message = message; // this is necessary if textView null
       TextView textView = (TextView) findViewById(R.id.dmax_spots_title);
       if (textView != null) {
            textView.setText(message);
       }
}
```

**Comment by d-max** [comment]
Cool, thanks

**Comment by ishaan-khan** [comment]
I believe "cool, thanks" suggests you have implemented this already. Why am I still getting this crash?

**Comment by d-max** [comment]
@ishaan-khan as you can see this issue is still open

**Comment by aistomin** [comment]
@d-max I've created a fix for this issue: #27 Could you please check it and accept the PR if the implemented logic is acceptable. Thanks.

**Comment by d-max** [comment]
@aistomin Thank you for the fix 👍
